config/locales/translations/zendesk_apps_support.yml

```diff
@@ -254,6 +254,10 @@ parts:
       key: "txt.apps.admin.error.app_build.translation.missing_required_key"
       title: "App builder job: required key missing from translation file"
       value: "Missing required key from %{file}: %{missing_key}"
+  - translation:
+      key: "txt.apps.admin.error.app_build.translation.missing_required_key_on_leaf"
+      title: "App builder job: required key missing from translation file. example: https://zendesk.box.com/s/m6yr2j0vai0a7qkimj2gcqiy61jznmn4"
+      value: "Missing required key %{missing_key} on leaf %{leaf} from %{file}"
```

Comment:
for the string. Always better to create separate pr for strings only @hiroiguchi

Reply:
@mariasica Noted. Thank you for your advice.

@fbvilela Thank you for uploading an image. can you add `screenshot: "https://zendesk.app.box.com/s/m6yr2j0vai0a7qkimj2gcqiy61jznmn4"` between title and value please?

Reply:
all done